changelog.md

```diff
@@ -4,10 +4,11 @@
 
 ### 0.14.2
 
-* Moved `@types/chalk` from dependencies to devDependencies - orta
+* Replaced deprecated `lodash.isarray` package with `Array.isArray` - damassi
```

Comment:
Ah, I see -- `danger` said to update the changelog and so I followed the format as it was there. It might be good to provide a little more direction in terms of version numbers, changelog, etc.